### ISSUE TITLE
JSON Error

### ISSUE THREAD

**Issue by jetwitaussi** [ushu/cordova-plugin-googleplaces#6]
Hi, I'm trying to use autocomplete feature on android but java code returns invalid json

```
Error: JSON error
    at f (GooglePlaces.dist.js:2)
    at Object.callbackFromNative (cordova.js:295)
    at processMessage (cordova.js:1119)
    at processMessages (cordova.js:1142)
```
Cordova callback returns error so i think there is an error on java code, didn't inspect it yet

**Comment by nagendra-d** [comment]
Have you enabled Places API for Android SDK & IOS SDK ?

**Comment by jetwitaussi** [comment]
Yes i have activated Places API for Web, Places SDK for Android, Places SDK for iOS
I don't see any activity in google console, and still get js error using `cordova.plugins.GooglePlaces.autocompleteQuery()`

I just perform autocomplete query with a string, i'm not using bounds or filter. If i use those parameters i have another error: `wrong arguments for autocompleteQuery(query, bounds, filter)`
Due to java code testing `args.length() > 1` on line 116, as with bounds and filter there are 3 parameters...

**Comment by lolaswift** [comment]
same error
@jetwitaussi have you fixed it?
